### ISSUE TITLE
Organization update does not work

### ISSUE THREAD

**Issue by Yoshiji** [auth0/auth0-cli#467]
### Description

The `orgs update` command does not work. The payload sent to the API includes the Organization's ID, which is not allowed: https://auth0.com/docs/api/management/v2/#!/Organizations/patch_organizations_by_id
The Organization's ID must be present in the URL, not in the body of the request.

### Reproduction

```
$ auth0 orgs update <id> -b "#BADA55"
=== optable-dev.us.auth0.com error
 ▸    400 Bad Request: Payload validation error: 'Additional properties not allowed: id'.
 ```

### Environment

- **Version of this library used:** latest `main` version (currently [7b0d17c](https://github.com/auth0/auth0-cli/commit/7b0d17c56b5aeb670336077d6aa22203e39082cd))
- **Version of the platform or framework used, if applicable:** N/A
- **Other relevant versions (language, server software, OS, browser):** N/A
- **Other modules/plugins/libraries that might be involved:** N/A

### How to fix

Omitting the Organization's ID when building the request payload in `updateOrganizationCmd` fixes the issue. I can make a PR out of it, but I am unsure if this is how you would like to fix it.

```
diff --git a/internal/cli/organizations.go b/internal/cli/organizations.go
index 80f85dd..66479f9 100644
--- a/internal/cli/organizations.go
+++ b/internal/cli/organizations.go
@@ -323,7 +323,7 @@ auth0 orgs update <id> -d "My Organization" -m "KEY=value" -m "OTHER_KEY=other_v
                        // re-hydrated by the SDK, which we'll use below during
                        // display.
                        o := &management.Organization{
-                               ID:          current.ID,
+                               // ID:          current.ID,
                                DisplayName: &inputs.DisplayName,
                        }
```

**Comment by willvedd** [comment]
This issue has been fixed in #583. However, it's gated behind our `v1` branch and won't be released until our first major version in the next few weeks. Will report back and close this issue once those releases become available.

**Comment by willvedd** [comment]
**Update:** We've included fix for this in the [v1.0.0-beta.1](https://github.com/auth0/auth0-cli/releases/tag/v1.0.0-beta.1) release. For those who are so inclined, we'd be happy to receive feedback on the beta before it goes GA, including feedback on this particular issue. Otherwise, we plan to close this issue out once v1.0.0 reaches GA. Again, appreciate everyone's patience!

**Comment by willvedd** [comment]
Fixed and released in [v1.0.0](https://github.com/auth0/auth0-cli/releases/tag/v1.0.0)
